docs/development.md

```diff
@@ -0,0 +1,33 @@
+# Developing Faabric
```

Comment:
We can add most of the `README` content into this `development.md` file as it's quite verbose now. Then the README can just be a short bit of blurb on what the project is with a few links.